### DISCUSSION TITLE
Would you mind we copy the example project to SeaORM?

### DISCUSSION THREAD

**Discussion by tyt2y3** [loco-rs/loco#391] [General]
Maintainer of SeaORM here. Thank you for the great work! Would you mind if we copy the example project to the SeaORM repo (so that it will be part of our CI), we'll also be sure to showcase it in README!

Btw, the `loco_rs::schema::*;` functions are brilliant!

**Comment by jondot** [comment]
Hi @tyt2y3 , sure of course go ahead. We want to say thanks for the great work on SeaORM! no other ORM comes close to it in developer experience, fantastic! ❤️

**Reply by billy1624** [reply]
Hey @jondot, I wonder if there is a html page / UI where it list all notes / data?

**Reply by jondot** [reply]
Hi, maybe try this? https://github.com/loco-rs/todo-list

**Reply by billy1624** [reply]
Perfect! I added the Loco todo list example on https://github.com/SeaQL/sea-orm/pull/2092

**Reply by kaplanelad** [reply]
thanks @billy1624 @tyt2y3

**Comment by billy1624** [comment]
Hey @jondot, thanks! Would you like to check the PR?
- https://github.com/SeaQL/sea-orm/pull/2080

**Reply by jondot** [reply]
looks great!
